### PR TITLE
2 packages from arakur/satysfi-actuarial at 0.0.1

### PR DESCRIPTION
**Pull request by arakur**
A package for actuarial notations.
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] Add to snapshot `snapshot-develop` :: satysfi-actuarial-doc.0.0.1 satysfi-actuarial.0.0.1
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6--1`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-7` :: satysfi-actuarial-doc.0.0.1 satysfi-actuarial.0.0.1
- [x] Add to snapshot `snapshot-stable-0-0-8` :: satysfi-actuarial-doc.0.0.1 satysfi-actuarial.0.0.1

### CODE REVIEW

File: packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Documentation of satysfi-actuarial"
+description: """
+Documentation of satysfi-actuarial.
+"""
+maintainer: "Arakur arakur65536@gmail.com"
+authors: "Arakur arakur65536@gmail.com"
```
Comment (na4zagin3) [inline packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam:7]:
```suggestion
maintainer: "Arakur <arakur65536@gmail.com>"
authors: "Arakur <arakur65536@gmail.com>"
```

File: packages/satysfi-actuarial/satysfi-actuarial.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Halo system implementation for actuarial notations in SATySFi"
+description: """
+Halo system implementation for actuarial notations in SATySFi.
+"""
+maintainer: "Arakur arakur65536@gmail.com"
+authors: "Arakur arakur65536@gmail.com"
```
Comment (na4zagin3) [inline packages/satysfi-actuarial/satysfi-actuarial.0.0.1/opam:7]:
```suggestion
maintainer: "Arakur <arakur65536@gmail.com>"
authors: "Arakur <arakur65536@gmail.com>"
```

File: packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+license: "MIT" # Choose what you want
+homepage: "https://github.com/arakur/satysfi-actuarial"
+dev-repo: "git+https://github.com/arakur/satysfi-actuarial.git"
+bug-reports: "https://github.com/arakur/satysfi-actuarial/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "0.0.7" }
```
Comment (na4zagin3) [inline packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam:13]:
```suggestion
  "satysfi" { >= "0.0.6" & < "0.1" }
```

File: packages/satysfi-actuarial/satysfi-actuarial.0.0.1/opam [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+license: "MIT"
+homepage: "https://github.com/arakur/satysfi-actuarial"
+dev-repo: "git+https://github.com/arakur/satysfi-actuarial.git"
+bug-reports: "https://github.com/arakur/satysfi-actuarial/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "0.0.7" }
```
Comment (na4zagin3) [inline packages/satysfi-actuarial/satysfi-actuarial.0.0.1/opam:13]:
```suggestion
  "satysfi" { >= "0.0.6" & < "0.1" }
```

File: packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam (hunk 1 above)
Comment (na4zagin3) [reply packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam:7]: It's better to enclose an email address with `<` and `>`.

File: packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam (hunk 3 above)
Comment (na4zagin3) [reply packages/satysfi-actuarial-doc/satysfi-actuarial-doc.0.0.1/opam:13]: Hmm, can we relax the upper bound if there's a reason not to support SATySFi 0.0.8 and later?

**Review by na4zagin3 — COMMENTED** [review]
Thank you for sending a package!  Let me fix some lines.

**Comment by na4zagin3** [comment]
Merging.  Thank you!

**Comment by na4zagin3** [comment]
Well, is this package incompatible with the latest azmath?

**Comment by na4zagin3** [comment]
Let me have a look at it by tomorrow.

**Comment by arakur** [comment]
Thank you.
Could I suppose the latest azmath is the one for version 0.0.3, which can be installed via satyrographos, and not be equal to the unreleased one at the repository of satysfi-azmath, right?
By the way, I found the azmath requirement was totally unnecessary and so will remove it.

**Comment by na4zagin3** [comment]
Thank you for updating the package!

Speaking of the incompatibility, it was my misunderstanding of the warning `inconsistent-snapshot +satysfi-azmath`, which was caused by azmath not having added to snapshots 0.0.6-1 to develop.  I fixed that with #518 and #519.  Sorry for confusion..

**Comment by arakur** [comment]
I see about azmath. Thank you for your supports.

**Comment by na4zagin3** [comment]
Thank you for your patience.  Merged!

**Pull request merged**